project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html

```diff
@@ -0,0 +1,358 @@
+<div>
+  <mat-accordion>
+    <mat-expansion-panel class="user-card" *ngFor="let user of usersData.content">
+      <mat-expansion-panel-header (click)="getUerData(user)">
+        <div class="float-left">
+          <div class="initialcircle display-inline-block margin-right-s">
+            <!-- <span>AB</span> -->
+            <ws-widget-avatar-photo [name]="getUseravatarName(user)" [photoUrl]="user?.photo" size="m">
+            </ws-widget-avatar-photo>
+          </div>
+          <div class="username display-inline-block" *ngIf="user.profileDetails && user.profileDetails.personalDetails">
+            <div class="mat-subheading-2 display-inline-block margin-remove">
+              {{user.profileDetails.personalDetails.firstname}}
+              <span class="status-label margin-left-s verified">Verified <mat-icon>check_circle</mat-icon></span>
+            </div>
+            <div class="mat-body-2 ws-mat-black40-text">Product Manager</div>
+          </div>
+          <div class="username display-inline-block" *ngIf="!user.profileDetails">
+            <div class="mat-subheading-2 display-inline-block margin-remove">
+              {{user.firstName}}
+              <span class="status-label margin-left-s verified" *ngIf="currentFilter !== 'nonverified'">Verified
+                <mat-icon>check_circle</mat-icon></span>
+              <span class="status-label margin-left-s nonverified" *ngIf="currentFilter === 'nonverified'">Non-verified
+                <mat-icon>check_circle</mat-icon></span>
+            </div>
+            <div class="mat-body-2 ws-mat-black40-text">Product Manager</div>
+          </div>
```

Comment:
Need to be divide into two component one for accordian and second with profile details

Reply:
Noted it, will check & update in next deployment in edit flow changes